prime-angular-frontend/src/app/shared/components/college-certification-form/college-certification-form.component.ts

```diff
@@ -10,6 +10,7 @@ import { ViewportService } from '@core/services/viewport.service';
 import { FormUtilsService } from '@core/services/form-utils.service';
 import { CollegeLicenceClass } from '@shared/enums/college-licence-class.enum';
 import { EnrolmentService } from '@enrolment/shared/services/enrolment.service';
+import { NursingLicenseCode } from '@shared/enums/nursing-license-code.enum';
```

Comment:
We try and group the imports based on module as much as possible.  It makes it easier to scan the dependencies when they are ordered by module.